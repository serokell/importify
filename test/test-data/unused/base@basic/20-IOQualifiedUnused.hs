module IOQualifiedUnused where

import qualified System.IO as IO (putStrLn)

main :: IO ()
main = pure ()
