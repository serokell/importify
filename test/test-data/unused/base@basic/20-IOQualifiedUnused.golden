module IOQualifiedUnused where


main :: IO ()
main = pure ()
